gwctl/pkg/printer/namespace.go

```diff
@@ -0,0 +1,60 @@
+package printer
+
+import (
+	"fmt"
+	"io"
+
+	"sigs.k8s.io/gateway-api/gwctl/pkg/policymanager"
+	"sigs.k8s.io/gateway-api/gwctl/pkg/resourcediscovery"
+	"sigs.k8s.io/yaml"
+)
+
+type NamespacesPrinter struct {
+	Out io.Writer
+}
+
+type namespaceDescribeView struct {
+	Name                     string                 `json:",omitempty"`
+	Labels                   map[string]string      `json:",omitempty"`
+	Annotations              map[string]string      `json:",omitempty"`
+	Status                   string                 `json:",omitempty"`
+	DirectlyAttachedPolicies []policymanager.ObjRef `json:",omitempty"`
+}
+
+func (nsp *NamespacesPrinter) PrintDescribeView(resourceModel *resourcediscovery.ResourceModel) {
```

Comment:
Lets add a test for this as well. (You can use some other tests like those for gateway and httproute for examples on how they can be written)

Reply:
Yes I was planning to add that in a seperate PR, but I will do it in this one PR itself now.